install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,254 @@
+[[install-config-aggregate-logging-sizing]]
+= Aggregate Logging Sizing Guidelines 
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The xref:../install_config/aggregate_logging.adoc#overview[ElasticSearch, Fluentd and Kibana] (EFK) stack aggregates logs from nodes and applications running inside your {product-title} installation.
+Once deployed it uses link:http://www.fluentd.org/architecture[Fluentd] to aggregate event logs from all nodes, projects and pods into link:https://www.elastic.co/products/elasticsearch[Elasticsearch]. It also provides a centralized link:https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] web UI where users and administrators can create rich visualizations and dashboards with the aggregated data.
+
+Fluentd link:https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html[bulk uploads] logs to an index, in json format, and ElasticSearch then routes your search requests to the appropriate shards.
+
+[[install-config-aggregate-logging-sizing-guidelines-installation]]
+== Installation
+The general procedure for installing an aggregate logging stack in {product-title} is described in xref:../install_config/aggregate_logging.adoc[Aggregating Container Logs].
```

Comment:
We need a to have an ID at the end of the file path, even if we're not pointing to a subsection. In this case, we need to point users to the ID found at the top of the page:

xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]